src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const { name,
+    age,
+    sex,
+    isMarried,
+    partnerName } = person;
+
+  const partner = (sex === 'm')
+    ? 'wife'
+    : 'husband';
+
+  const status = (isMarried)
+    ? `${partnerName} is my ${partner}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">{status}</p>
+    </section>
```

Comment:
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
      
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
      <p className="Person__partner">
        {status}
      </p>
    </section>
```

Reply:
@nurdanokcu  Shoul I move the whole function to Person.scss file?